MathProblems/HammingDistance.py

```
@@ -0,0 +1,22 @@
+               while len(y)<len(x):
+                   y=['0']+y 
+          count=0
+          index=0 
+          while index<len(y):
+                if x[index]!=y[index]:
+                    count+=1
+                index+=1
+          print(x)
+          print(y)
+          return count
+Solution().hammingDistance(4,1)
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 0

(E305)